components/brave_vpn/resources/panel/components/loading-panel/style.ts

```diff
@@ -2,42 +2,80 @@
 // This Source Code Form is subject to the terms of the Mozilla Public
 // License, v. 2.0. If a copy of the MPL was not distributed with this file,
 // you can obtain one at https://mozilla.org/MPL/2.0/.
-import styled from 'styled-components'
 
-export const Box = styled.div`
-  width: 100%;
-  height: 100%;
-  background: ${(p) => p.theme.color.panelBackground};
-  overflow: hidden;
-  min-height: 260px;
-`
+import styled from 'styled-components'
+import Icon from '@brave/leo/react/icon'
+import ProgressRing from '@brave/leo/react/progressRing'
+import vpnSvgUrl from '../../assets/svg-icons/vpn-icon.svg'
 
 export const PanelContent = styled.section`
   display: flex;
-  align-items: center;
-  justify-content: center;
   flex-direction: column;
-  padding: 20% 24px 25px 24px;
+  align-items: center;
+  padding: 24px;
+  position: relative;
   z-index: 2;
 `
 
-export const LoaderIconBox = styled.div`
-  width: 64px;
-  height: 64px;
-  margin-bottom: 24px;
+export const SettingsButton = styled.button`
+  position: absolute;
+  right: 16px;
+  top: 16px;
+  border: 0;
+  padding: 0;
+  background: transparent;
+  cursor: pointer;
+`
+
+export const VpnLogo = styled.span`
```

Comment:
Is there a reason the `Nala` icon isn't working for this? (maybe it's something we need to fix in Nala!)

Reply:
Ah weird - I take it `vpn-shield-color` isn't working either?

https://nala.bravesoftware.com/?path=/story/components-icon--default&args=name:vpn-shield-color

Did you add it to the `leo_icons` array?

Reply:
Hmm, maybe I was wrong before. it works well with `Icon`.